### PR TITLE
fix usePost

### PR DESCRIPTION
**Pull request by pivilartisant**
Fix use post where we get no params

### CODE REVIEW

**Review by Pierregvx — APPROVED** [review]
LGTM

**Review by Thykof — APPROVED** [review]

File: web/massastation/src/custom/api/usePost.ts [hunk 1]
```diff
@@ -28,9 +28,10 @@ export function usePost<TBody, TResponse = null>(
     mutationFn: async ({ params, payload }) => {
       const queryParams = new URLSearchParams(params).toString();
       if (queryParams) {
-        url.concat(`?${queryParams}`);
+        url = `${url}?${queryParams}`;
```
Comment (maxime-carabina) [inline web/massastation/src/custom/api/usePost.ts:31]: `.concat` is better than doing this 🤔

Comment (Pierregvx) [reply web/massastation/src/custom/api/usePost.ts:31]: it didn't work with xoncat

Comment (pivilartisant) [reply web/massastation/src/custom/api/usePost.ts:31]: this does not work but if you want we can do this : 
```
url = url.concat(`?${queryParams}`);
```

Comment (Thykof) [reply web/massastation/src/custom/api/usePost.ts:31]: I think that `    url = `${url}?${queryParams}`;` is easier to read that concat

Comment (maxime-carabina) [reply web/massastation/src/custom/api/usePost.ts:31]: > this does not work but if you want we can do this :
> 
> ```
> url = url.concat(`?${queryParams}`);
> ```

Thank you! Better on this way

**Pull request merged**